### DISCUSSION TITLE
Can we find a dictionnary of 'BT' code and json fields ?

### DISCUSSION THREAD

**Discussion by MathildeDI** [OP-TED/eForms-SDK#772] [Q&A] [answered]
Hello,
I'm looking for a list of BT code, their meaning, and the corresponding field in the xml/json file. 
For instance BT-500 = PartyName.
Thanks,
Mathilde

**Comment by YvesJo** [comment, marked as answer]
Hi,
You'll be able to find some eforms related information at https://single-market-economy.ec.europa.eu/single-market/public-procurement/digital-procurement/eforms_en
Also the excel spreadsheet of the Implementing Regulation Annex at https://ec.europa.eu/docsroom/documents/52697
In fields.json, you may find, for each defined Field, the identifier of the associated Business Term:

>  {
>     **"id" : "BT-27-Lot",**
>     "parentNodeId" : "ND-LotValueEstimate",
>     "name" : "Estimated Value",
>     **"btId" : "BT-27"**,

Field IDs are derived from the BT IDs.

In the translations folder of the SDK you may find the labels of the BTs (file business-term_XX.xml) and the labels for the fields (file field_xx.xml).
KR
